docs/android/v2/plugins/Virtual-background.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Virtual Background
+nav: 12
+---
+
+## Introduction
+
+Virtual background plugin helps in customising one’s background i.e. replacing the background with a static image.
+This guide provides an overview of usage of the virtual background plugin of 100ms.
+
+![VirtualBackground](/docs/v2/vb.jpg)
+
+## Supported Android API Levels
+
+Min api version for the virtual background plugin is 21. Same as the min api version core sdk for 100ms. The lowest SDK version it can work with is 21. It is built for armeabi-v7a, arm64-v8a, x86, and x86_64 architectures.
+
+## Prerequisites
+
+Familiarity with Android Studio and the [fundamentals](https://developer.android.com/training/basics/firstapp) of android apps.
+
+## Common Terms
+
+-   `image background` - The image which the plugin should replace the background with in the user's video.
+-   `plugin load time` - The time taken by the plugin to load the  machine learning model for the first time. The machine learning model is what will identify what part of the image the background is and is essential to virtual background working.
+
+## Add dependency
+
+-   Add the 100ms Virtual Background plugin and sdk dependency to your app-level `build.gradle` [![](https://jitpack.io/v/100mslive/android-sdk.svg)](https://jitpack.io/#100mslive/android-sdk).
+
+```json{3,4}:build.gradle
+dependencies {
+ // See the version in the jitpack badge above.
+ implementation 'com.github.100mslive:android-sdk-virtual-background:x.x.x'
+ implementation 'com.github.100mslive:android-sdk:x.x.x'
+    }
+```
+## Interfaces
+
+### Instantiate VirtualBackground
+
+Instantiate the 100ms Virtual Background plugin like this:
+
+<Tabs id="instanstiate-VirtualBackground" items={['Kotlin', 'Java']} />
+
+<Tab id='instanstiate-VirtualBackground-0'>
+
+```kotlin
+      val hmsSdk = HMSSDK
+              .Builder(application)
+              .build()
+      val virtualBackground = HMSVirtualBackground(hmsSdk, imageBitmap)
+```
+
+</Tab>
+
+<Tab id='instanstiate-VirtualBackground-1'>
+
+```java
+   HMSSDK hmssdk = new HMSSDK
+                           .Builder(application)
+                           .build();
+   HMSVirtualBackground virtualBackground = new HMSVirtualBackground(hmssdk, imageBitmap)
+```
+</Tab>
+
+That's it. You have instanstiated virtual background plugin successfully. 🥳
```

Comment:
Make virtual background or Virtual Background (caps) consistent throughout the doc